luminance/src/blending.rs

```diff
@@ -76,3 +76,27 @@ pub struct Blending {
   /// Destination factor.
   pub dst: Factor,
 }
+
+/// Blending configuration to represent combined or separate options.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum BlendOption {
+  /// blending with combined RGBA
+  Combined(Blending),
+
+  /// blending with RGB and alpha separately
+  Separate(Blending, Blending),
```

Comment:
Here, we need to make the variant more explicit. Something along the lines of:

```suggestion
  Separate {
    rgb: Blending,
    alpha: Blending
  },
```